components/organisms/animations/AnimFromScratchCodeblock.vue

```diff
@@ -57,16 +57,16 @@
               <span>&lt;</span>
               <span class="text-red-400">h1</span>
               <span>&gt;</span>
-              Hello world !
+              <span>Hello world !</span>
```

Comment:
This fixes some extra spacing in the animation:
![image](https://user-images.githubusercontent.com/3385679/125176679-9d920780-e192-11eb-977e-435af644de95.png)